dev-libs/ancient/ancient-2.0.0.recipe

```diff
@@ -0,0 +1,76 @@
+SUMMARY="A collection of old decompression routines"
+DESCRIPTION="This is a collection of decompression routines for various old formats that \were populare on Amiga, Atari computers and other old systems. A simple \command-line application is provided aswell."
```

Comment:
DESCRIPTION line up to 100 characters, see: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#description